doc/dcf.md

```diff
@@ -26,4 +26,8 @@ Those Gen3 commands are used to generate data refresh reports, augmented manifes
 
 * `gen3 dcf redaction DR16`
 
-  Generate redaction report for data release 16
\ No newline at end of file
+  Generate redaction report for data release 16
+
+* `gen3 dcf create-google-bucket gdc-target-phsxxx-controlled phsxxx False`
```

Comment:
False without specifying --public might be a little confusing. Can we change it to public/protected in gen3 cmd to make it more clear?

Reply:
I changed to `public/controlled`